client/src/ui/molecules/breadcrumbs.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import { DocParent } from "../../document/types";
+
+import "./breadcrumbs.scss";
+
+export default function Breadcrumbs({ parents }: { parents: DocParent[] }) {
```

Comment:
I know it's already been done in a couple of places, but try to avoid default exports and stick to regular exports instead. 
The only good reason for using a default export is when a component needs to be lazy-loaded.

Reply:
Another item to add to the design system 👍